specfile/tags.py

```diff
@@ -463,6 +463,17 @@ def __init__(
         self._separator = separator
         self.comments = comments.copy()
 
+    def __eq__(self, other: object) -> bool:
+        if not isinstance(other, Tag):
+            return NotImplemented
+        return (
+            self.name == other.name
+            and self.value == other.value
+            and self._expanded_value == other._expanded_value
+            and self._separator == other._separator
+            and self.comments == other.comments
```

Comment:
Do we consider `Tag` to be mutable? Wouldn't it make sense to implement also `__hash__()`, like in `Macro`?
See: [https://docs.python.org/3.11/reference/datamodel.html#object.__hash__](https://docs.python.org/3.11/reference/datamodel.html#object.__hash__).

Reply:
I think we should consider them mutable. This also corresponds to how the `tags()` context manager in `Specfile` operates -- you can modify a value of a tag and then the change is mapped back to the low-level content. Whereas with macros, it doesn't make much sense to change anything.